deploy/roles/devtools6/tasks/main.yml

```diff
@@ -0,0 +1,32 @@
+---
+- name: Install devtools repo
+  template:
+    src: devtools-6.repo.j2
+    dest: /etc/yum.repos.d/devtools-6.repo
+    mode: 0644
+  when: not custom_repo
+  tags:
+    - devtools
+
+- name: Copy devtools shell script
+  template:
+    src: devtools-6.sh.j2
+    dest: /opt/devtools-6.sh
+    mode: 0755
+  when: not custom_repo
```

Comment:
@rioh We don't need the conditional here, since the script should be copied in both Vagrant and the actual environment.